sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -94,34 +106,44 @@ private void BackoffConcluded()
         /// This State Machine uses delegates for the callback action and the backoff evaluation.
         /// These delegates are updated whenever a state is changed.
         /// </summary>
-        /// <param name="cancellationToken"></param>
-        private void Run(CancellationToken cancellationToken)
+        private void Run()
         {
-            while (!cancellationToken.IsCancellationRequested)
+            try
             {
-                var callbackStarted = DateTimeOffset.UtcNow;
-
-                _callbackAction.Invoke();
-
-                var timeSpentInThisTick = DateTimeOffset.UtcNow - callbackStarted;
-
-                TimeSpan nextTick;
-
-                // Check if we need to backoff.
-                if (_evaluateBackoff.Invoke())
+                while (true)
                 {
-                    Debug.WriteLine($"{DateTime.Now}: Backing off.");
-                    SetBackoffState();
-                    nextTick = _period;
+                    var callbackStarted = DateTimeOffset.UtcNow;
+
+                    _callbackAction.Invoke();
```

Comment:
nit: Not sure if you did this intentionally but FYI you can just invoke the delegate directly (`_callbackAction.Invoke();` -> `_callbackAction();`). One below too.

Reply:
I think I've seen `.Invoke()` more often, or maybe I recognize it more often than the alternative. :)